### PR TITLE
FailoverClustering Bulk Fixup (1/4) - Line formatting updates

### PR DESCRIPTION
**Pull request by robinharwood**
Bulk reflowed text and header formatting update before applying further quality updates.

This will be followed by separate PRs for 
	1. Text formatting updates
		a. Bulk formatting checklist update before applying further quality updates:
			- cmdlet name - Change bolds to code spans
			- cmdlet/vars - Change bar ecmdlet/var to code spans
			- parameters - change italic formatting to bold
			- parameters - change code span formatting to bold
			- change italics to use underscores
			- change bold-underscores to asterisk
			- change bold keystrokes to kbd
			- Use contractions
			- Fix indented code blocks
			- Find bad syntax reflows
	2. Update example formatting
		a. Bulk updating example formatting before content updates

### CODE REVIEW

**Comment by robinharwood** [comment]
Hi @michaeltlombardi / @sdwheeler,

I believe this PR is release ready if you would be kind enough to review and if appropriate sign-off/merge 😊

The main update was to reflow and fix any line return issues.

Please let me know if you have any questions or if I need to apply any other updates.

Thanks Robin

**Review by michaeltlombardi — CHANGES_REQUESTED** [review]
I went through for a style/formatting pass, there's a few places where it seems like text got munged and some broken formatting.

That being said, in case you want to focus this PR (and the others) on _only_ cleaning up the line lengths, I'll review the other PRs only for broken formatting and similar issues.

I haven't done a thorough prose editing here, either, but could do in the future. This pass was lightweight, addressing formatting where I thought it would likely also affect line length.

File: docset/winserver2022-ps/failoverclusters/Get-Cluster.md [hunk 1]
```diff
@@ -111,35 +117,42 @@ cluster3
...
 QuarantineThreshold : 3
 ```
 
-This example shows default values for QuarantineThreshold and QuarantineDuration for the local cluster.
+This example shows default values for QuarantineThreshold and QuarantineDuration for the local
+cluster.
 
- -- QuarantineThreshold: This is the number of times that a node can become isolated in an hour before the cluster will be quarantined. This is set to 3 by default.
+ -- QuarantineThreshold: This is the number of times that a node can become isolated in an hour
+ before the cluster will be quarantined. This is set to 3 by default.
```
[the remark this answers is not in the document]
Comment (robinharwood) [reply docset/winserver2022-ps/failoverclusters/Get-Cluster.md:147]: Hey @michaeltlombardi - yes, I will address in #3162 and other follow ups.

File: docset/winserver2022-ps/failoverclusters/Get-Cluster.md [hunk 2]
```diff
@@ -111,35 +117,42 @@ cluster3
...
 
- --QuarantineDuration: This setting, set to 7200 seconds or 2 hours by default, controls how long a host will remain quarantined.
+ --QuarantineDuration: This setting, set to 7200 seconds or 2 hours by default, controls how long a
+ host will remain quarantined.
```
[the remark this answers is not in the document]
Comment (robinharwood) [reply docset/winserver2022-ps/failoverclusters/Get-Cluster.md:150]: Hey @michaeltlombardi - yes, I will address in #3162 and other follow ups.

Edit: Note, that #3162 need to be rebased once this is live :-)

File: docset/winserver2022-ps/failoverclusters/Add-ClusterGenericApplicationRole.md [hunk 3]
```diff
@@ -99,8 +110,10 @@ Accept wildcard characters: False
 ```
 
 ### -CommandLine
+
 Specifies the Windows PowerShell® command line to use for the highly available generic application.
```
[the remark this answers is not in the document]
Comment (robinharwood) [reply docset/winserver2022-ps/failoverclusters/Add-ClusterGenericApplicationRole.md:114]: Hey @michaeltlombardi - yes, I will address in #3162 and other follow ups.

Edit: Note, that #3162 need to be rebased once this is live :-)

File: docset/winserver2022-ps/failoverclusters/Add-ClusterNode.md [hunk 4]
```diff
@@ -16,22 +16,26 @@ Adds a node (server) to a failover cluster.
...
 
-Before adding the new node, you should run validation tests on the existing nodes together with the proposed new node.
-By running the validation tests, you can confirm that the server to be added is connected correctly to the networks and storage and that it contains the same software updates.
+The **Add-ClusterNode** cmdlet adds a node, or server, to a failover cluster. Before adding the new
+node, you should run validation tests on the existing nodes together with the proposed new node.
 
-Note: This cmdlet cannot be run remotely without Credential Security Service Provider (CredSSP) authentication on the server computer.
+Before adding the new node, you should run validation tests on the existing nodes together with the
+proposed new node. By running the validation tests, you can confirm that the server to be added is
+connected correctly to the networks and storage and that it contains the same software updates.
```
[the remark this answers is not in the document]
Comment (robinharwood) [reply docset/winserver2022-ps/failoverclusters/Add-ClusterNode.md:30]: Hey @michaeltlombardi - I think this is GitHub doing a strange comparison, if I look at the file directly, I think it looks OK. I want to make sure you are happy too though :-)

[the remark this answers is not in the document]
Comment (michaeltlombardi) [reply docset/winserver2022-ps/failoverclusters/Add-ClusterNode.md:30]: Okay! Happy to merge as-is then, there's going to be a bunch more PRs that will catch anything along the way anyhow.

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**